README.md

```
@@ -39,62 +39,83 @@ project-supplied index files are modified to produce the corrected index  files
+<!--
 The `metadata-tools` module is available via the `rms-metadata-tools`[![image](https://raw.githubusercontent.com/SETI/rms-metadata-tools/main/icons/link.png)](https://pypi.org/project/rms-metadata-tools)
 package on PyPI and can be installed with:
 
 ```sh
 pip install rms-metadata-tools
 ```
+-->
+
+Clone the rms-metadata-tools repo, activate the venv, and install the dependencies:
+
+% git clone https://github.com/SETI/rms-metadata-tools.git
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)